alert-common/src/main/java/com/synopsys/integration/alert/common/security/EncryptionUtility.java

```diff
@@ -113,7 +113,11 @@ private String getEncodedSalt() {
     }
 
     public boolean isPasswordSet() {
-        return null != getPassword();
+        return StringUtils.isNotBlank(getPassword());
```

Comment:
This is a good change, but we should double check that we are clear about blank values not being allowed for these settings in the doc.